cmake/CMakeLists.txt

```diff
@@ -526,6 +527,29 @@ if(PKG_USER-SCAFACOS)
   include_directories(${SCAFACOS_INCLUDE_DIRS})
 endif()
 
+if(PKG_USER-PLUMED)
+  option(DOWNLOAD_PLUMED "Download Plumed (instead of using the system's one)" OFF)
+  if(DOWNLOAD_PLUMED)
+    include(ExternalProject)
+    ExternalProject_Add(plumed_build
+      URL https://github.com/plumed/plumed2/releases/download/v2.4.3/plumed-2.4.3.tgz 
+      URL_MD5 bed9c88c8c635b269c1cd4a0598a2934
+      BUILD_IN_SOURCE 1
+      CONFIGURE_COMMAND <SOURCE_DIR>/configure --prefix=<INSTALL_DIR> 
+                                               $<$<BOOL:${BUILD_SHARED_LIBS}>:--with-pic>                                                  )
+    ExternalProject_get_property(plumed_build INSTALL_DIR)
+    set(PLUMED_INSTALL_DIR ${INSTALL_DIR})
+    list(APPEND LAMMPS_DEPS plumed_build)
+    list(APPEND LAMMPS_LINK_LIBS ${PLUMED_INSTALL_DIR}/lib/libplumed.so)
```

Comment:
I just realized this will break if LAMMPS gets installed as `libplumed.so` doesn't get installed alongside with it.

Reply:
You should link against `-lplumedWrapper -ldl`, which is what the conventional make build does. Please see `src/USER-PLUMED/Install.sh`. This way, the LAMMPS executable is usable without libplumed.so being installed where it can be found by the dynamic linker, only using fix plumed will fail.

Reply:
The issue will stay the same as `libplumedWrapper.so` also doesn’t get installed either!

Reply:
And I will do that change once you assign it to me ;-)